### DISCUSSION TITLE
Enable max_background_jobs

### DISCUSSION THREAD

**Discussion by xiaobiaozhao** [apache/kvrocks#1284] [General]
Use `max_background_jobs` instead of `max_background_compactions` and `max_background_flushes`

https://github.com/facebook/rocksdb/blob/main/include/rocksdb/options.h#L699

**Comment by torwig** [comment]
Since `max_background_compactions` and `max_background_flushes` are deprecated, I think it's a good idea to replace them with `max_background_jobs`.
